lib/grpc_kit/calls/client_bidi_streamer.rb

```diff
@@ -18,13 +18,13 @@ def initialize(*)
 
       # @param data [Object] request message
       # @return [void]
-      def send_msg(data)
+      def send_msg(data, last: false)
```

Comment:
Why is this flag needed? I think that calling BidiStreamer#close_and_send at the end of the stream is a correct way when using bidi_streamer like below.
https://github.com/cookpad/grpc_kit/blob/bc5180e4d05c68c3f85a823418951ef2e22bbb0c/spec/integrations/bidi_streamer_spec.rb#L75

Doesn't it work this way?

Reply:
Fixed 67629b5. It works 😃